### ISSUE TITLE
NTR: Gastric biopsy: CMV virus present (Lab: endoscopy and exploratory surgery)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8662]
New term request Gastric biopsy: CMV virus present (nadea3_200221153622):

**Comment by pnrobinson** [comment]
@MickeySegal
is this intended to be a PCR finding?

**Comment by MickeySegal** [comment]
I've changed our finding to "Gastric biopsy: CMV intranuclear inclusions by in-situ hybridization" with citation https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3760702/

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001005
- label: Positive gastric CMV intranuclear inclusions
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: PMID:24010116
- def: Demonstration of the presence of cytomegalovirus in the stomach by means of a method such as in-situ hybridization.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8662
- synonyms:
